packages/psss/src/utils/auth.js

```diff
@@ -5,22 +5,20 @@
 
 import { AccessPolicy } from '@tupaia/access-policy';
 
-export const checkIsAuthorisedForMultiCountry = user =>
-  getPermittedEntitiesForUser(user).length > 1;
+export const checkIsAuthorisedForMultiCountry = user => getEntitiesAllowedByUser(user).length > 1;
 
 export const checkIsAuthorisedForCountry = (user, match) =>
-  getPermittedEntitiesForUser(user).some(entityCode => entityCode === match.params.countryCode);
+  getEntitiesAllowedByUser(user).some(
+    entityCode => entityCode.toLowerCase() === match.params.countryCode,
+  );
 
-export const getPermittedEntitiesForUser = user => {
+export const getEntitiesAllowedByUser = user => {
   if (!user) {
     return [];
   }
 
-  // Todo: Update with the correct access policy check
-  // @see: https://github.com/beyondessential/tupaia-backlog/issues/1268
-  const accessPolicy = new AccessPolicy(user.accessPolicy);
-
-  // For testing...
-  return ['as', 'to'];
-  // return ['to'];
+  const entities = new AccessPolicy(user.accessPolicy).getEntitiesAllowed('PSSS');
+  // Todo: filter this list to countries only
+  console.log('entities allowed', entities);
```

Comment:
Remove. Currently we only put countries into it, and this assumption is all around the codebase, so it's fine to have that here as well